contracts/test/v0.8/dev/KeeperRegistryDev.test.ts

```diff
@@ -140,55 +155,81 @@ describe('KeeperRegistryDev', () => {
       .connect(owner)
       .deploy(9, linkEth)
     transcoder = await upkeepTranscoderFactory.connect(owner).deploy()
+    mockArbGasInfo = await mockArbGasInfoFactory.connect(owner).deploy()
+    mockOVMGasPriceOracle = await mockOVMGasPriceOracleFactory
+      .connect(owner)
+      .deploy()
+
+    const arbOracleCode = await ethers.provider.send('eth_getCode', [
+      mockArbGasInfo.address,
+    ])
+    await ethers.provider.send('hardhat_setCode', [
+      '0x000000000000000000000000000000000000006C',
```

Comment:
I think this is honestly a pretty slick solution for tests. I'm a touch concerned about other cases, ex what do we do if we run arbitrum / optimism locally? Is this going to make it more difficult down the road to mock this when we're not using hardhat? Would vote to inject the address as a constructor arg - it's a very tried and true method to avoid testing challenges later.